`ExpectedNonexistentPathError` on `ki push` of `gsse-anki` deck

Cause unknown. Unable to reproduce this locally.

**Edit.** This happened because `.ki/` was *not* in the `.gitignore` file.

See below.

```python
Run ki push
  
/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/prettyprinter/__init__.py:325: UserWarning: Failed to import 'attrs' PrettyPrinter extra. If you don't need it, call install_extras with exclude=['attrs']
  warnings.warn(


( ●    ) Initializing stage repository...

(  ●   ) Initializing stage repository...

(   ●  ) Initializing stage repository...

(    ● ) Initializing stage repository...

(     ●) Initializing stage repository...

(    ● ) Initializing stage repository...

(   ●  ) Initializing stage repository...

(  ●   ) Initializing stage repository...

( ●    ) Initializing stage repository...

(●     ) Initializing stage repository...

( ●    ) Initializing stage repository...

(  ●   ) Initializing stage repository...

(   ●  ) Initializing stage repository...

(    ● ) Initializing stage repository...

(     ●) Initializing stage repository...

(    ● ) Initializing stage repository...

(   ●  ) Initializing stage repository...

(  ●   ) Initializing stage repository...

( ●    ) Initializing stage repository...

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.6/x64/bin/ki", line 8, in <module>
    sys.exit(ki())
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/click/core.py", line 30, in __call__
    return self.main(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "<@beartype(ki.push) at 0x7f5b6a29ef80>", line 30, in push
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/ki/__init__.py", line 2227, in push
    head_kirepo: KiRepo = copy_kirepo(head, f"{HEAD_SUFFIX}-{md5sum}")
  File "<@beartype(ki.copy_kirepo) at 0x7f5b6a477be0>", line 49, in copy_kirepo
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/ki/__init__.py", line 235, in copy_kirepo
    raise ExpectedNonexistentPathError(ki_dir)
ki.types.ExpectedNonexistentPathError: Expected this path not to exist, but it does: '/tmp/tmpf9gclbb1/ki-
head-884d12b330f65c599083d404baa32a87/.ki'
(●     ) Initializing stage repository...

Error: Process completed with exit code 1.
```

--- a/ki/types.py
+++ b/ki/types.py
@@ -323,10 +323,17 @@ class StrangeExtantPathError(Exception):
 class ExpectedNonexistentPathError(FileExistsError):
     @beartype
     def __init__(self, path: Path, info: str = ""):
-        msg = f"""
+        top = f"""
         Expected this path not to exist, but it does: '{path}'{info.rstrip()}
         """
-        super().__init__(errwrap(msg))
+        msg = """
+        If the path is to the `.ki/` metadata directory, this error may have
+        been caused by a `.gitignore` file that does not include `.ki/` (this
+        metadata should not be tracked by git). Check if this pattern is
+        included in the `.gitignore` file, and if it is not included, try
+        adding it.
+        """
+        super().__init__(f"{top}\n\n{errwrap(msg)}")
 
 
 class NotKiRepoError(Exception):

